Gems/ImGui/Code/Source/ImGuiManager.h

```diff
@@ -24,6 +24,8 @@
 #include <AzCore/Module/DynamicModuleHandle.h>
 #endif // defined(LOAD_IMGUI_LIB_DYNAMICALLY)  && !defined(AZ_MONOLITHIC_BUILD)
 
+namespace AZ { class JobCompletion; }
```

Comment:
[coding standards](https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md#positioning-of-curly-braces): curly braces should be on it's own line